Makefile.cache

```diff
@@ -71,6 +71,8 @@ SONIC_COMMON_FLAGS_LIST :=  $(CONFIGURED_PLATFORM) \
                             $(CONFIGURED_ARCH) \
                             $(BLDENV) \
                             $(SONIC_DEBUGGING_ON) \
+                            $(SONIC_VERSION_CONTROL_COMPONENTS) \
```

Comment:
Makefile.cache runs before pip/apt-get. If source code and dependencies not changed, it will copy package from local path. This feature shouldn't change Makefile.cache.

Reply:
Fixed it.